lxd-agent/events.go

```diff
@@ -32,7 +35,7 @@ func (r *eventsServe) String() string {
 func eventsSocket(d *Daemon, r *http.Request, w http.ResponseWriter) error {
 	typeStr := r.FormValue("type")
 	if typeStr == "" {
-		typeStr = "logging,operation,lifecycle"
+		typeStr = "logging,operation,lifecycle,config"
```

Comment:
Why is that needed?
That feels wrong since we'll never hit eventsGet for a `config` event type

Reply:
When listening on `/dev/lxd/sock` for a config change, we need this here. Otherwise the user won't get notified.

Reply:
Oh, I see, we're abusing eventsGet from within devlxd :)

Reply:
@monstermunchkin can you add a comment in there so we remember why this is done that way?